resource/reactivedrop_swedish.txt

```diff
@@ -618,7 +618,7 @@
 "rd_hoiaf_new_personal_best_score_on_this_mission"		"%s1 har förbättrat sin säsongs bästa poäng på detta uppdrag från %s2 till %s3, som ökar dess bonus poäng med +%s4!\n"
 // %s1 is a player name; %s2 and %s3 are numbers.
 "[english]rd_hoiaf_bounty_mission_bonus_claimed"		"By completing this mission, %s1 has claimed a bounty of %s2 points! %s1 has claimed %s3 points from mission bounties so far this season.\n"
-"rd_hoiaf_bounty_mission_bonus_claimed"		"Genom att klara av detta uppdrag, %s1 har samlat in en belöning på %s2 poäng! %s1 har samlat in %s3 poäng från uppdrag denna säsong.\n"
+"rd_hoiaf_bounty_mission_bonus_claimed"		"Genom att klara av detta uppdrag, har %s1 samlat in en belöning på %s2 poäng! %s1 har samlat in %s3 poäng från uppdrag denna säsong.\n"
```

Comment:
This one is incorrect, because the har (has) cant be before the bounty amount.

Reply:
Because %s1 is the name of the player, so it would be "har (namn) samlat...) Its supposed to be %s1 har samlat...

Reply:
If you could get that reverted and change the other things mentioned